src/System.ServiceModel.Security/tests/ServiceModel/ISecurityCapabilitiesTest.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using Infrastructure.Common;
+using Xunit;
+
+public static class ISecurityCapabilitiesTest
+{
+    [WcfFact]
+    public static void SecurityCapabilities_HttpsTransportBindingElement()
+    {
+        ISecurityCapabilities capability = CreateISecurityCapabilities(new HttpsTransportBindingElement());
+
+        Assert.Equal("EncryptAndSign", capability.SupportedRequestProtectionLevel.ToString());
+        Assert.Equal("EncryptAndSign", capability.SupportedResponseProtectionLevel.ToString());
+        Assert.Equal("False", capability.SupportsClientAuthentication.ToString());
```

Comment:
```suggestion
        Assert.False(capability.SupportsClientAuthentication);
```

Reply:
And similarly there's an Assert.True you can use too.

Reply:
Thanks for the feedback, PR updated.